### DISCUSSION TITLE
why does WinMerge always highlight the text "ods(" ?

### DISCUSSION THREAD

**Discussion by CipherBruceH** [WinMerge/winmerge#2276] [Q&A] [answered]
Why does WinMerge always highlight the text "ods(" ?

I'm using 64-bit portable latest version on Win10 and Win11.
No plugins enabled.
No line filters enabled.

I use Windows OutputDebugString() a lot in my code and I have a macro defined as ods() - and WinMerge always highlights "ods(", no matter its context. Have text like "aods(xyz" and the embedded "ods(" part is highlighted.

No harm - just curious.
Thanks,
CipherBruceH

**Comment by sdottaka** [comment]
`ods(` may be registered in the marker window
The marker window can be displayed via Edit → Marker menu item.

![image](https://github.com/WinMerge/winmerge/assets/98126/809332d7-3fa9-495f-a9c3-c32b3b74dffa)

**Reply by CipherBruceH** [reply]
Interesting... There were no Markers, but Markers were [enabled]. After disabling Markers, "ods(" was no longer highlighted.

So, something odd going on with the Marker code... including the fact that "ods(" was the only string effected. If I enter this in a blank WinMerge page - only the "ods(" strings light up... (when my blank Markers are [enabled]):
ods(
abc(
ods(
xyz(

Additionally, if I actually create any Marker, as soon as I [enable] Markers... the "ods(" occurences get highlighted - (yellow highlighter).

Does this happen to anyone else?
Thanks!

**Reply by sdottaka** [reply, marked as answer]
Could you please press Ctrl+F and search for something?
If the highlight of `ods(` disappears, it is simply a highlight that matches the search.

**Reply by CipherBruceH** [reply]
That is it @sdottaka.
WinMerge just remembers whatever was [Find]ed last and highlights that string until you search for something different!
Even if you exit WinMerge and relaunch it - you get any hits highlighted from whatever your last [Find] was.

Thanks!
